### PR TITLE
global: add identifiers tab in documents frontsite and display isbn in backoffice

### LINKED ISSUE #722: Frontsite: Don't display "Related" if there are none
The front site's document details page, in the "Details" tab, should not display "Related" if there are no related records.

### LINKED ISSUE #723: Frontsite: Add identifiers tab to document details
Identifiers tab is missing for document details page on front site.
There is such a tab on the backoffice.
 - [x] decide how to display the identifiers (maybe the tab is not a good solution, maybe they should be somewhere else?), decide how to represent them to be understandable (maybe the backoffice way will also work good for the frontsite?
- [x] implement the identifiers presentation component (in a similar structure as the backoffice)
- [x] add alternative identifiers to identifiers tab in backoffice

### PR DESCRIPTION
**Pull request by KonstantinaStoikou**
* Add message "No related documents" in document details tab on frontsite, closes #722
* Add ISBN, DOI in document details tab on frontsite
* Add identifiers tab in document details page on frontsite
* Display alternative identifiers in document details page on backoffice
* Fix error in series identifiers tab in details page on frontsite (not all identifiers were displayed)

Closes #723

Document backoffice details page:
document_back_details

Document frontsite details page:
document_front_details
Selection_001

Series backoffice details page:
series_back_details

Series frontsite details page:
series_front_details

### CODE REVIEW

**Review by ntarocco — APPROVED** [review]

File: ui/src/pages/frontsite/components/Document/DocumentInfo.js [hunk 1]
```diff
@@ -23,6 +24,49 @@ export class DocumentInfo extends Component {
     return null;
   }
 
+  renderISBN() {
+    const identifiersISBN = this.metadata.alternative_identifiers
+      ? this.metadata.alternative_identifiers.filter(
+          identifier => identifier.scheme === 'ISBN'
```
Comment (kpsherva) [inline ui/src/pages/frontsite/components/Document/DocumentInfo.js]: Could you move the ISBN to be in the identifiers? I know it is now in the  select under `alternative_identifiers` but it was not intended to be, as it is specific to the domain, therefore not something additional but rather crucial.
In order to do that you will have to change one of the vocabularies JSON files to move it to `identifiers`, similarly for series.
As a results this field will be taken here from the `identifiers` in the end, like the DOI below

File: ui/src/pages/frontsite/components/Identifiers/Identifiers.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ export const IdentifierRows = ({ includeSchemes, identifiers }) => {
     // Only include whitelisted schemes if includeSchemes is set
     if (!includeSchemes || includeSchemes.includes(id.scheme)) {
       const value = { value: id.value, material: id.material };
-      if (id.scheme in identifiers) {
+      if (id.scheme in idsByScheme) {
```
Comment (kpsherva) [inline ui/src/pages/frontsite/components/Identifiers/Identifiers.js:34]: are the `alternative_identifiers` displayed correctly in the frontsite in the end?

Comment (KonstantinaStoikou) [reply ui/src/pages/frontsite/components/Identifiers/Identifiers.js:34]: yes

**Review by kpsherva — APPROVED** [review]

File: ui/src/pages/frontsite/components/Document/DocumentRelations.js [hunk 3]
```diff
@@ -87,10 +88,16 @@ export class DocumentRelations extends Component {
       return (
         <>
           <Divider horizontal>Related</Divider>
-          {this.renderMultiparts()}
-          {this.renderSerials()}
-          {this.renderLanguages()}
-          {this.renderEditions()}
+          {isEmpty(this.relations) ? (
```
Comment (ntarocco) [inline ui/src/pages/frontsite/components/Document/DocumentRelations.js]: can you please remove this modification? it is already fixed and improved in my PR for both Document and Series :)

Comment (ntarocco) [reply ui/src/pages/frontsite/components/Document/DocumentRelations.js]: See here: https://github.com/inveniosoftware/invenio-app-ils/pull/789 (screenshot)

**Pull request merged**